README.md

```diff
@@ -91,18 +91,32 @@ After that you have the following instance methods available to you.
     
     `TelephoneNumber.parse("3175082237", :US)`
     
+    If you pass an E164 formatted number, we will determine the country on the fly. 
+    
+    `TelephoneNumber.parse("+13175082237")`
+    
   - #### `valid?`
     
     Returns boolean value indicating whether or not a particular number is valid.
     
     `TelephoneNumber.valid?("3175082237", :US) ==> true`
     
+    If you are looking to validate against a specific set of keys, you can pass in an array of keys
+    
+    `TelephoneNumber.valid?("3175082237", :US, [:mobile, :fixed_line]) ==> true`
```

Comment:
The formatting looks weird here, but that's more of a nitpick than anything:

<img width="868" alt="mobi_telephone_number_at_akfernun-patch-1" src="https://cloud.githubusercontent.com/assets/13686303/25747539/cdd45aaa-3175-11e7-9c3d-db5d662052d4.png">